lhotse/cut.py

```diff
@@ -1594,6 +1716,38 @@ def truncate(
             else None,
         )
 
+    # noinspection PyUnusedLocal
+    def extend_by(
+        self,
+        *,
+        duration: Seconds,
+        direction: str = "both",
+        preserve_id: bool = False,
+        **kwargs,
```

Comment:
This method lacks documentation. Also, it is not clear what `kwargs` can contain.

Reply:
I mostly just copied this from the `truncate()` method for PaddingCut. But you're right --- I'll add some documentation, and remove kwargs since I don't think they're required.